README.rst

```diff
@@ -6,10 +6,6 @@ Protego
    :target: https://pypi.python.org/pypi/protego
    :alt: Supported Python Versions
 
-.. image:: https://img.shields.io/travis/scrapy/protego/master.svg
-   :target: https://travis-ci.org/scrapy/protego
-   :alt: Build Status
```

Comment:
Shall we replace it by the GitHub Actions one?  Based on the Scrapy one, I imagine it is something like:

```
.. image:: https://github.com/scrapy/protego/workflows/CI/badge.svg
   :target: https://github.com/scrapy/protego/actions?query=workflow%3ACI
   :alt: CI
```